commcare_connect/opportunity/tests/test_views.py

```
@@ -539,6 +542,108 @@ def test_org_member_can_resend_invite(
+
+        url = reverse(
+            "opportunity:fetch_attachment", args=(organization.slug, visit.opportunity.id, blob_meta.blob_id)
+        )
+        client.force_login(org_user_member)
+
+        response = client.get(url)
+        assert response.status_code == 200
+        storage_handler_getitem_mock.assert_called_once()
+
+    @mock.patch.object(StorageHandler, "__getitem__")
+    def test_user_cannot_fetch_managed_opp(self, storage_handler_getitem_mock, org_user_member, organization, client):
```

Ruff (0.14.5)

Unused method argument: `storage_handler_getitem_mock`

(ARG002)